UPGRADING.rst

```diff
@@ -10,6 +10,35 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#4673 Eliminate burner accounts
+===============================
+
+When the operator completes the following steps, they should announce for other
+developers to do the same.
+
+#) Remove your burner account from the Google Cloud project:
```

Comment:
F

<img width="1034" alt="image" src="https://user-images.githubusercontent.com/5143256/214721265-4c134fac-aca0-4f9d-ae35-0f99048b3d4b.png">

Reply:
Still broken. Note the quoted block depicted in the screenshot I attached.

Reply:
I initially thought you might've also been referring to the quoted blocks but disregarded it [since there was precedent.](https://github.com/DataBiosphere/azul/blob/develop/UPGRADING.rst#4224-eliminate-personal-service-accounts) for them.

Reply:
Since they are obviously (in my opinion) a mistake, feel free to fix both.